### ISSUE TITLE
How to proceed with model evaluation

### ISSUE THREAD

**Issue by YongSeongLee25** [tensorflow/models#10133]
Hello

You want to proceed with the evaluation in model_main_tf2.py.

python model_main_tf2.py --pipeline_config_path=/pipeline_d3.config --model_dir=model_570_1 --checkpoint_dir=model_570_1 --alsologtostderr

I understand that if you execute this code, you will proceed with the evaluation.
Just like in training, we print out the loss.
Map, Precision, recall, etc. are not visible.
Can you tell me why?

**Comment by Chitti21** [comment]
@saikumarchalla @YongSeongLee25 
Hi I am also interested in knowing about this. 

I guess --check_point flag takes the latest checkpoint. is there an option to : 
1. get map at any desired check point
2. check the map directly at desired iou (like --map 0.5iou or --map 0.25iou and so on) 
3. check the map at desired iou and desired conf_thresh
4. calculate the best map obtained during the entire training session and how to export the best model for further use
5. view the map during the training process

**Comment by Chitti21** [comment]
@saikumarchalla @ymodak any suggestions?

**Comment by vighneshbirodkar** [comment]
Can you share your logs with the eval code ?

**Comment by YongSeongLee25** [comment]
@vighneshbirodkar 
#10171 
This error occurs when you run it again.
